genomeuploader/ena.py

```diff
@@ -275,22 +288,29 @@ def handle_genomes_registration(self, sample_xml, submission_xml, webin, passwor
         receiptXml = minidom.parseString((submissionResponse.content).decode("utf-8"))
         receipt = receiptXml.getElementsByTagName("RECEIPT")
         success = receipt[0].attributes["success"].value
-        if success == "true":
-            aliasDict = {}
+        aliasDict = {}
+        try:
             samples = receiptXml.getElementsByTagName("SAMPLE")
             for s in samples:
                 sraAcc = s.attributes["accession"].value
                 alias = s.attributes["alias"].value
                 aliasDict[alias] = sraAcc
-        elif success == "false":
+            logger.info(f'{len(aliasDict)} genome samples successfully registered.')
+        except:
+            logger.info("No newly registered samples")
+        # check errors and search for existing accessions
+        if success == "false":
             errors = receiptXml.getElementsByTagName("ERROR")
```

Comment:
Is the error an unstructured string? Or does it have a format, something we could parse with the xml module?

Reply:
It will print error below in the script